### PR TITLE
Disable download state button if disconnected

### LINKED ISSUE #1310: Downloading app state should not be possible while disconnected
## Steps to Reproduce
1. Go to https://lightclient.raiden.network/
2. Click on identicon
3. Click on Back up state
4. Click on Download State

## Expected Result
Nothing happens, it should be disabled because there is no web3 provider
Maybe a hint/tooltip why downloading state is disabled

## Actual Result
Warning modal, which after confirm downloads a state file that contains `undefined`

### PR DESCRIPTION
**Pull request by nephix**
Fixes #1310

### CODE REVIEW

File: raiden-dapp/tests/unit/views/backup-state.spec.ts [hunk 1]
```diff
@@ -44,6 +47,14 @@ describe('BackupState.vue', () => {
...
 
+  test('download state button disabled if disconnected', () => {
+    const downloadStateButton = wrapper.find(
+      '.backup-state__buttons__download-state'
+    );
+
+    expect(downloadStateButton.classes()).toContain('v-list-item--disabled');
+  });
+
   // test('upload state', () => {
```
Comment (kelsos) [inline raiden-dapp/tests/unit/views/backup-state.spec.ts:58]: I know it's unrelated to the PR but any idea why we have a commented test here?

Comment (nephix) [reply raiden-dapp/tests/unit/views/backup-state.spec.ts:58]: Probably the upload/download state ticket was too big and got split up
They will be implemented in #1308

**Comment by nephix** [comment]
Codecov's reporting doesn't make any sense here. I'm force-merging this

File: raiden-dapp/tests/unit/views/backup-state.spec.ts (hunk 1 above)
Comment (kelsos) [reply raiden-dapp/tests/unit/views/backup-state.spec.ts:58]: Ok but it would be nice to avoid commented code in the repo in the future :)

**Pull request merged**